### DISCUSSION TITLE
Same Application within a Group

### DISCUSSION THREAD

**Discussion by tweschke** [rxhanson/RectanglePro-Community#43] [Q&A]
Hi,
I'm trying to create within Applications a Group, which would set my current three Chrome browser windows to the same size and place when I use a shortcode, but it looks like I can't do that, as the second instance of Chrome is just placed in the same place as the first on in the group.

Is there a way I can do that? 

While I design, I need to have 2 or even 3 Chrome browsers open, which are set up as one main centre window and two additional windows (one to the left and one to the right).

Many thanks.

**Comment by rxhanson** [comment]
At the moment, there is no way to do this, but I am planning on adding this capability. The most straightforward implementation would be to just place windows in frontmost-backmost according to where the additional application instances are in the group. Ultimately, this would feel a little arbitrary as to which windows go where (I don't mentally keep track of the z-index of my 2nd & 3rd browser windows), so this would kind of be operating on the mentality that you are not overly concerned with which browser window goes where - you just want it tiled. Is this your expectation?

**Reply by tweschke** [reply]
Thank you for your quick reply. It would be great to see an option, on how we could set the same application within a group to different sections of the screen. This is important for me, as I work on a 49" monitor and it is no fun in dragging and resizing applications all day. At this stage, I have kind of worked around it by opening my main area as "Centre Half" and the outside section as "First and Last Fourths", but I shrink them a little, so that I can maybe also still have a code/text editor in the top corner.
Please find attached a screenshot of what I would like to do with a single group shortcut.

Many thanks
![design-layout-min](https://user-images.githubusercontent.com/65774908/160593906-e9e0365d-56d6-4e67-9a93-155377d67b83.png)
.

**Reply by rxhanson** [reply]
Thanks for the additional info. Do you care which browser window goes where? That's kind of the main limitation up front that I don't have a good idea for at the moment. It would be great if I could create some kind of distinguishing identification per browser window that could be configured in the app, but there's not a good way to do so that I'm currently aware of.

**Reply by sameer-git** [reply]
Yeah it'd be cool to have this feature.  In my case I need it for Zoom, where I want the speaker view window on my left screen and the sharing window on a specific section of the right screen
